desktop/core/src/desktop/js/apps/editor/components/sqlScratchpad/api.ts

```diff
@@ -0,0 +1,32 @@
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { get, post } from 'api/utils';
+
+const LOGOUT_URL = '/hue/accounts/logout';
+const LOGIN_URL = '/hue/accounts/login';
+const JWT_URL = 'iam/v1/get/auth-token/';
+
+export const login = async (username: string, password: string, jwt = true): Promise<void> => {
```

Comment:
`jwt = true` --> `auth = 'jwt` might be more explicit/evolvable as there are many other auths (e.g. email and not username, Google SSO...) (and `jwt` feels more like a token instead of a bool at first sight)

Should we also move this one out of SS itself, e.g. api/auth.js or utils? 


Quick Docs update?
https://github.com/cloudera/hue/blob/master/docs/docs-site/content/developer/components/scratchpad/_index.md

Reply:
True, will switch to auth=x and good idea to move to api/auth as quite generic.